app/_hub/kong-inc/aws-lambda/index.md

```diff
@@ -223,9 +223,9 @@ from being executed. We are planning to remove this limitation in the future.
 
 3. Now we need to create the lambda function itself, will do so in N.Virginia Region (code us-east-1).
 
-    In Lambda Management, create a new function Mylambda, there will be no blueprint as we are going to paste the code below; for the execution role let's choose an existing role specifically LambdaExecutor created previously
+    In Lambda Management, create a new function Mylambda, there will be no blueprint as we are going to paste the code below (which is an example code snippet); for the execution role let's choose an existing role specifically LambdaExecutor created previously.
```

Comment:
Maybe just "we are going to paste the sample code below" for simplicity? Or "example code".